src/app/services/api.service.ts

```diff
@@ -110,7 +114,7 @@ export class ApiService {
   //Executes a http post request to add a user to a project
   addUsertoProject(user_id) {
     return new Promise(resolve => {
-      var post_object = { user_id: user_id, scope: ["READ", "WRITE"], projectId: this.project.project["id"] }
+      var post_object = { userId: user_id, scope: [ "RIGHT_UNSPECIFIED"], projectId: this.project.project["id"] }
```

Comment:
I think RIGHT_UNSPECIFIED is not right here. It should probably [ "RIGHT_READ", "RIGHT_WRITE"]

Reply:
Changed in Commit 0fda10285f9c1d346588d468e8c50ec2bf3cf2df